src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/args/GcloudArgs.java

```diff
@@ -65,4 +62,13 @@
   public static List<String> get(Map<?, ?> keyValueMapping) {
     return Args.keyValues(keyValueMapping);
   }
+
+  /**
+   * @return list of args for the common arguments in {@link Configuration}.
+   */
+  public static List<String> get(Configuration configuration) {
```

Comment:
Added a test. Also swapped around the order for the assert params in some of the other tests in the class since I noticed the expected / actual values were in the wrong order.